enterprise_gateway/services/processproxies/container.py

```diff
@@ -191,6 +191,7 @@ def shutdown_listener(self):
 
     async def confirm_remote_startup(self) -> None:
         """Confirms the container has started and returned necessary connection information."""
+        self.log.debug("Trying to confirm kernel container startup status")
```

Comment:
Could we remove this please?

Reply:
it's debug log, and helps to try to understand the area that things are happening on issues that are hard to find